### ISSUE TITLE
Consider adding pghp to SeaORM showcase!

### ISSUE THREAD

**Issue by billy1624** [CHATALOT1/pghp#16]
Hey @CHATALOT1, feel free to submit a PR to showcase your project!
- https://github.com/SeaQL/sea-orm/issues/403

**Comment by CHATALOT1** [comment]
Hey @billy1624, thanks for the offer! Unfortunately, I've recently decided to [rewrite the backend/API of the project in Go](https://github.com/CHATALOT1/pghp/issues/15) due to a few reasons, including:
- Go seems like it may prove more useful as a programming language for future projects I plan to work on as opposed to Rust (as this project is mainly intended as a fun test-bed to help me get fully comfortable with a stack for future projects). This includes Go's potential for a much more stable approach to runtime plugins than Rust offers.
- I have struggled developing with SeaORM as it currently stands.

I plan to return to using Rust for this sort of project in the future, and wish you and the SeaQL team well for the future.
